HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -814,6 +858,15 @@ public PathByActionArguments(string action, string controller, RouteValueDiction
     }
     
     private static string GetLocalAuthorityPendingMessagePartialViewPath(Questionnaire questionnaire)
+    {
+        var partialViewName = questionnaire.CustodianCode switch
+        {
+            _ => "Default"

```

Comment:
My assumption is that it's entirely possible that there might be more specific messages in future - checking that's a sensible one?

(if not then I'd be in favour of scrapping the switch to keep things simple)

Reply:
Your assumption is correct :) when writing this I was asked to do so with that in mind